### PR TITLE
Merge Open science chapter into master

### PR DESCRIPTION
**Pull request by r-j-arnold**
### Summary

Now the open science chapter has been reviewed and merged into the open science branch this pull request merges that branch into master.

### List of changes proposed in this PR (pull-request)

Merge open science chapter into master

### What should a reviewer concentrate their feedback on?

@LouiseABowler Following you last comment on #175 I think you're right that a table of contents for each chapter would be helpful, but I think since the sprints so near it'd be good to confirm everyone's happy with that there, and then add it in another pull request.

As for the bibliography, I agree it's a mess but I'm leaving it for now as in the last zoom meeting it was deemed low priority and to be sorted out later.

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [humans.md](humans.md).

### CODE REVIEW

File: chapters/open_science.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+
+## What to learn next
+
+If you haven't had a chance already, take a look at the chapter on version control, particularly the sections on GitHub in the latter half.
+
+## Further reading
+
+[This](http://book.openingscience.org.s3-website-eu-west-1.amazonaws.com/) book on open science has a great deal of interesting information on open science. For information specific to open source software [this](https://opensource.guide/) is a good place to look. For more information on DOIs and citing resources look [here](http://www.doi.org/index.html). If you want to take a look at an active open source project this textbook *is* one. You can be found on GitHub [here](https://github.com/alan-turing-institute/the-turing-way) and/or take a look at the project [website](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)
+
+project
```
Comment (LouiseABowler) [inline chapters/open_science.md]: Is this a typo, or should it be a link/image?

File: chapters/open_science.md [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+- [ ] Keep notes in an Electronic Lab Notebook
+- [ ] Make your notebooks publicly accessible online.
+
+## What to learn next
+
+If you haven't had a chance already, take a look at the chapter on version control, particularly the sections on GitHub in the latter half.
+
+## Further reading
+
+[This](http://book.openingscience.org.s3-website-eu-west-1.amazonaws.com/) book on open science has a great deal of interesting information on open science. For information specific to open source software [this](https://opensource.guide/) is a good place to look. For more information on DOIs and citing resources look [here](http://www.doi.org/index.html). If you want to take a look at an active open source project this textbook *is* one. You can be found on GitHub [here](https://github.com/alan-turing-institute/the-turing-way) and/or take a look at the project [website](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)
```
Comment (LouiseABowler) [inline chapters/open_science.md]: Not sure we need to link to the Turing website here as it doesn't contain any of the book's content. Looks like the final sentence is unfinished; would something like "Head over to our [main page](https://github.com/alan-turing-institute/the-turing-way) to explore the rest of the project, and check out our [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) for details of how you can involved." be what you meant here?

**Comment by LouiseABowler** [comment]
@r-j-arnold, just one unfinished sentence to sort out and then this is good to merge - prod me on Gitter if I'm slow about approving when you're done. And yes, good call on leaving the table of contents discussion until the sprint; it'd be good to talk over with everyone else.

**Comment by r-j-arnold** [comment]
Dammit. Sorry @LouiseABowler, I'd mistakenly crossed fixing the further reading section off my list, I thought I'd done it. I have now, thanks for the catch. I've also made it clear the link to the website is for additional info about the project, not book material.

**Comment by r-j-arnold** [comment]
Great, thanks @LouiseABowler! Merging now.

**Pull request merged**